cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -224,7 +224,11 @@ metadata:
   labels:
     k8s-app: kube-dns
 spec:
-  replicas: 1
+  replicas: 2
```

Comment:
Why increase this value?

Reply:
Increased to 2 as initially suggested in coredns/deployment#13, as part of a rolling update strategy.

Reply:
If you do 2 you want inter-pod anti-affinity to make sure they are not on the same host.